test/bigtest/interactors/jobs-view.js

```diff
@@ -44,4 +44,8 @@ export default @interactor class JobsView {
   headerDropdown = new HeaderDropdown('[data-pane-header-actions-dropdown]');
   headerDropdownMenu = new HeaderDropdownMenu();
   confirmationModal = new ConfirmationModalInteractor();
+
+  whenLoaded() {
+    return this.when(() => this.isPresent).timeout(10000);
```

Comment:
Hmm....is this supposed to be something like

```isJobPane = isPresent('#pane-view-job')``` 

```whenLoaded() {
    return this.when(() => this.isJobPane).timeout(10000);
}```

Reply:
I don't think so. I cribbed it [from Coburn here](https://github.com/folio-org/ui-users/blob/master/test/bigtest/interactors/user-view-page.js#L32), and I use it successfully in the `jobs-view-test.js` tests

Reply:
Ah my bad...I missed the defaultScope at the top..so yeah, thats good